python/F3DPythonBindings.cxx

```diff
@@ -158,7 +152,9 @@ PYBIND11_MODULE(f3d, module)
     .def("setIcon", &f3d::window::setIcon,
       "Set the icon of the window using a memory buffer representing a PNG file")
     .def("setWindowName", &f3d::window::setWindowName, "Set the window name")
-    .def("getCamera", &f3d::window::getCamera, py::return_value_policy::reference);
+    .def("getCamera", &f3d::window::getCamera, py::return_value_policy::reference)
+    .def("getWorldFromDisplay", &f3d::window::getWorldFromDisplay, "Get world coordinate point from display coordinate")
+    .def("getDisplayFromWorld", &f3d::window::getDisplayFromWorld, "Get display coordinate point from world coordinate");
```

Comment:
missing getSize ?

Reply:
Done, but I changed the API to `getWidth` and `getHeight` because we can't handle argument by reference with pybind11, see https://github.com/pybind/pybind11/issues/134  
I think we may have the problem on other APIs.  
We need to improve our tests of the bindings

Reply:
this is concerning, lets open an issue to track this.